.github/workflows/benchmark.yml

```
@@ -11,7 +11,7 @@ jobs:
       matrix:
         node_version:
           - "22"
-    runs-on: self-hosted
+    runs-on: github-benchmark-runner
```

TIL you can't target by runner name, only by labels, so I created a new label for this runner...
```suggestion
    runs-on: equinix-bare-metal
```